src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
```

Comment:
Consider renaming the function to be more descriptive, for example, 'calculateTotalRentalCost' to clarify that it calculates the total cost.